runtime/help/plugins.md

```diff
@@ -62,6 +62,10 @@ that micro defines:
 * `onBufferOpen(buf)`: runs when a buffer is opened. The input contains
    the buffer object.
 
+* `onBufferOptionChanged(buf, option, old, new)`: runs when a option of the
+   settings of a buffer has changed. The input contains the buffer object,
+   the option, the old and the new value.
```

Comment:
"the option name"? If we want to make it that verbose anyway, let's at least make this verbosity useful.